projects/fsastorefrontlib/src/cms-components/my-account/quote/quote-details/quote-details.component.html

```diff
@@ -0,0 +1,205 @@
+          </div>
+        </li>
+        <li class="accordion-list-item row">
+          <div class="col-6 col-sm-5 item-label">
+            {{ 'fscommon.price' | cxTranslate }}
+          </div>
+          <div class="col-6 col-sm-5 value notice">
+            {{ quote.quotePrice }}
+          </div>
+        </li>
+        <li class="accordion-list-item row">
+          <div class="col-6 col-sm-5 item-label">
+            {{ 'fscommon.paymentFrequency' | cxTranslate }}
+          </div>
+          <div class="col-6 col-sm-5 value notice">
+            {{ quote.paymentFrequency }}
+          </div>
+        </li>
+      </ul>
+    </cx-fs-accordion-item>
+    <ng-container *ngIf="quote.insuredObjectList?.insuredObjects?.length > 0">
+      <cx-fs-accordion-item
+        [title]="'fscommon.whoOrWhatIsInsured' | cxTranslate"
+      >
+        <ng-container
+          *ngFor="let insuredObject of quote.insuredObjectList.insuredObjects"
+        >
+          <ng-container
+            *ngIf="
+              insuredObject.insuredObjectItems &&
+              insuredObject.insuredObjectItems.length > 0
+            "
+          >
+            <ul class="accordion-list container-fluid">
+              <li
+                *ngIf="
+                  getTranslation(
+                    quote.defaultCategory?.code,
+                    insuredObject.insuredObjectType | lowercase
+                  ) as insuredObjectType
+                "
+                class="accordion-list-item row sub-title"
+              >
+                <div class="col-12">
+                  {{ insuredObjectType }}
+                </div>
+              </li>
+              <ng-container
+                *ngFor="
+                  let insuredObjectItem of insuredObject.insuredObjectItems
+                "
+              >
+                <li
+                  *ngIf="
+                    getTranslation(
+                      quote.defaultCategory?.code,
+                      insuredObjectItem.label
+                    ) as insuredObjectItemLabel
+                  "
+                  class="accordion-list-item row"
+                >
+                  <div class="col-6 col-sm-5 item-label">
+                    {{ insuredObjectItemLabel }}
+                  </div>
+                  <div class="col-6 col-sm-5 value notice">
+                    {{ insuredObjectItem.value }}
+                  </div>
+                </li>
+              </ng-container>
+            </ul>
+          </ng-container>
+        </ng-container>
+      </cx-fs-accordion-item>
+    </ng-container>
+    <ng-container *ngIf="cart$ | async as cart">
+      <cx-fs-accordion-item [title]="'fscommon.whatsIncluded' | cxTranslate">
+        <ul
+          class="accordion-list container-fluid"
+          *ngIf="cart.entries.length > 0"
+        >
+          <li
+            *ngFor="
+              let entry of cart.entries[0].product.price.oneTimeChargeEntries
```

Comment:
Maybe add `?` operator after `cart.entries[0]` to ensure the object doesn't throw an error. Perhaps do a check for this on all objects in this template.

Reply:
Done!